src/commcare_cloud/ansible/deploy_commcarehq.yml

```
@@ -73,22 +73,22 @@
   tags: services
 
 - name: Management Command Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
   tasks:
     - include_vars: roles/commcarehq/vars/main.yml
     - include_tasks: roles/commcarehq/tasks/management_commands.yml
   tags: services
 
 - name: Prometheus Supervisor Config
-  hosts: commcarehq
+  hosts: commcarehq:formplayer
```

This is only needed if there are HQ processes running on the machine.
```suggestion
  hosts: commcarehq
```